### PR TITLE
Add svs recipe of 'Oniku'

### PR DESCRIPTION
**Pull request by South-Twilight**

### CODE REVIEW

File: egs2/oniku/svs1/local/data.sh [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+mkdir -p ${ONIKU}
+
+train_set="tr_no_dev"
+train_dev="dev"
+recog_set="eval"
+
+if [ ${stage} -le 0 ] && [ ${stop_stage} -ge 0 ]; then
+    log "stage 0: Data Download"
+    # The Oniku data should be downloaded from ??
```
Comment (ftshijt) [inline egs2/oniku/svs1/local/data.sh:38]: could you update the link?

File: egs2/oniku/svs1/local/prep_segments.py [hunk 2]
```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python3
```
Comment (ftshijt) [inline egs2/oniku/svs1/local/prep_segments.py:1]: can we have possibilities to use the same one as ofuton as they are very similar?

File: egs2/oniku/svs1/local/prep_segments_from_xml.py [hunk 3]
```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env python3
```
Comment (ftshijt) [inline egs2/oniku/svs1/local/prep_segments_from_xml.py:1]: Ditto

File: egs2/oniku/svs1/run.sh [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+# training and inference configuration
+train_config=conf/train.yaml
+inference_config=conf/decode.yaml
+
+# text related processing arguments
+g2p=pyopenjtalk
+cleaner=none
+
+#vocoder path
+vocoder_path="/data3/tyx/vocoder/train_nodev_oniku_hifigan.v1/checkpoint-298828steps.pkl"
```
Comment (ftshijt) [inline egs2/oniku/svs1/run.sh:39]:
```suggestion
```

File: egs2/oniku/svs1/run.sh [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+    --cleaner ${cleaner} \
+    --train_config "${train_config}" \
+    --inference_config "${inference_config}" \
+    --train_set "${train_set}" \
+    --valid_set "${valid_set}" \
+    --test_sets "${test_sets}" \
+    --score_feats_extract "${score_feats_extract}" \
+    --srctexts "data/${train_set}/text" \
+    --ngpu 1 \
+    --vocoder_file "${vocoder_path}"
```
Comment (ftshijt) [inline egs2/oniku/svs1/run.sh:63]:
```suggestion
```

**Comment by ftshijt** [comment]
Also, could you add an entry at `egs2/README.md` and `egs2/TEMPLATE/svs1/db.sh` for Oniku?

File: egs2/oniku/svs1/local/prep_segments.py (hunk 2 above)
Comment (South-Twilight) [reply egs2/oniku/svs1/local/prep_segments.py:1]: They're the same.

File: egs2/oniku/svs1/local/prep_segments_from_xml.py (hunk 3 above)
Comment (South-Twilight) [reply egs2/oniku/svs1/local/prep_segments_from_xml.py:1]: They're the same. Maybe can use soft-link to use without copying.

**Pull request merged**